packages/uswds-core/src/js/start.js

```diff
@@ -15,13 +13,16 @@ const svg4everybody = require("./polyfills/svg4everybody");
 
 uswds.components = components;
 
-domready(() => {
+const initComponents = () => {
   const target = document.body;
   Object.keys(components).forEach((key) => {
     const behavior = components[key];
     behavior.on(target);
   });
   svg4everybody();
-});
+};
+
+window.addEventListener("DOMContentLoaded", initComponents, { once: true });
 
-module.exports = uswds;
+exports.default = uswds;
+exports.initComponents = initComponents;
```

Comment:
uswds is the default export, but adding the `initComponents` function in case we want to re-trigger component initialization in the future.

For example: if we want to add a custom event for Storybook loading (different than DOM ready) and re-initialize _all_ components.